src/web/js/trove/world-lib.js

```diff
@@ -750,10 +759,20 @@
                 k2();
               } else {
                 if (extras.closeWhenStop) {
-                  if (extras.closeBigBangWindow) {
-                    extras.closeBigBangWindow();
+                  if (lif) {
+                    runtime.ffi.throwMessageException('Cannot have both last-image and close-when-stop(true)');
+                  } else {
+                    if (extras.closeBigBangWindow) {
+                      extras.closeBigBangWindow();
+                    }
+                    Jsworld.shutdownSingle({cleanShutdown: true});
                   }
-                  Jsworld.shutdownSingle({cleanShutdown: true});
+                } else if (lif) {
+                  var handler = lif();
```

Comment:
daa7a18 (and the commit before it) includes the fixes. The last-image handler wasn't handing its canvas node on to its wrapper's success continuation. Also, made the handler's user code execute inside a safeCall.

Here's a perhaps slightly more illustrative mod of your example program, to show that the last image is not the last `to-draw` image.

```
include world
include image

big-bang(10,
  [list:
    last-image(lam(x): text(num-to-string(75), 75, "red") end),
    to-draw(lam(x): text(num-to-string(x), x, "blue") end),
    on-tick(lam(x): x + 10 end),
    stop-when(lam(x): x > 90 end)])
```